docs/developer-guide/project-conventions.md

```diff
@@ -79,14 +77,34 @@ We follow a loose version of the [Git flow branching model](https://nvie.com/pos
 - Release branches should always be based on: `develop`
 - Release branches should be merged via PR into `main`, followed by PR to merge `main` into `develop`.
 
+**See below for merging guidance**
+
 ### Naming branches
 
-- Branch names for ticketed new features should start with: `feature/<JIRA-TICKET-with-short-description>`.
-- Branch names for ticketed bugs should start with: `fix/<JIRA-TICKET-with-short-description>`.
-- Branch names for release should start with: `release/<major.minor.patch>`.
-- Branch names for housekeeping tasks or other unticketed work should start with: `chore/<short-description>`
-- Ticket numbers should be included in branch names wherever possible. For example:
-  - `feature/DF-123-extra-squiggles`
-  - `fix/DF-999-image-view-error`
-  - `chore/short-description`
-- Stick with alphanumeric characters and hyphens where possible and avoid random special characters.
+- Use only alphanumeric characters and hyphens where possible and avoid special characters.
+- Branch names for ticketed new features should follow: `feature/JIRA-TICKET-NUMBER-with-short-description`
+- Branch names for ticketed bug fixes should follow: `fix/JIRA-TICKET-NUMBER-with-short-description`
+- Branch names for releases should follow: `release/major.minor.patch`
+- Branch names for housekeeping tasks or other unticketed work should follow: `chore/short-description`
+- For example:
+    - `feature/UN-123-extra-squiggles`
+    - `fix/DF-999-image-view-error`
+    - `release/1.0.0`
+    - `chore/update-documentation`
+
+### Naming pull requests
+
+- Pull requests for features and bug fixes should be titled: `JIRA-TICKET-NUMBER: short-description`
+- Pull requests for release branches should be titled: `Release X.X.X into main`
```

Comment:
One more to add (new) - When merging main to develop, the default title `Release <x.x.x> into main (#<a number>)` appears and we edit it. How would we want to update it to indicate merging from main to develop but also retain the default message.

Suggested
```
- When merging `main` to `develop`, default pull requests title should be suffix  with `-merge main to develop`

Ex-default name is `Release <x.x.x> into main (#<PR number>)`, then updated name is `Release <x.x.x> into main (#<PR number>) - merge main to develop`
```
I think after merging, this will then appears as `Release <x.x.x> into main (#<PR number>) - merge main to develop (#<CURRENT PR number)`

We don't have a fixed one, but can follow something now onwards. Any thoughts?